lib/rpush/client/active_model/apns/device_token_format_validator.rb

```diff
@@ -4,7 +4,7 @@ module ActiveModel
       module Apns
         class DeviceTokenFormatValidator < ::ActiveModel::Validator
           def validate(record)
-            return if record.device_token =~ /^[a-z0-9]\w+$/i
+            return if record.device_token =~ /\A[a-z0-9]\w+\z/i
```

Comment:
Could you provide a reference for this change? Is this just personal preference or does something force us to do it?

Reply:
Yes, this comes out of [ActiveModel::Validations::FormatValidator](https://github.com/rails/rails/blob/5cfd58bbfb8425ab1931c618d98b649bab059ce6/activemodel/lib/active_model/validations/format.rb#L38-L42).